src/layout/actions/layout.rs

```diff
@@ -224,6 +254,36 @@ impl LayoutTree {
         self.validate();
     }
 
+    /// Gets the active container and toggles it based on the following rules:
+    /// * If horizontal, make it vertical
+    /// * else, make it horizontal
+    /// This method does *NOT* update the actual views geometry, that needs to be
+    /// done separately by the caller
+    pub fn toggle_active_horizontal(&mut self) {
```

Comment:
hmmmm `toggle_cardinal_tiling`? Or something to the effect to mean switching between "up and down" (vertical) and "left and right" (horizontal)? I don't use "cardinal" anywhere else though, so it could be confusing, but that's the best name I can come up with.